pyproject.toml

```diff
@@ -84,6 +84,75 @@ dp-accounting = [
     "scipy>=1.7.1"
 ]
 
+dev = [
+    "flake8",
+    "pylint",
+    "pytype",
+    "pytest",
+    "pytest-xdist",
+    "typing_extensions",
+]
+
 [tool.setuptools.packages.find]
 include = ["README.md", "LICENSE"]
 exclude = ["*_test.py"]
+
+[tool.pytype]
+inputs = ["optax"]
+disable = ["import-error"]
+keep_going = true
+
+[tool.pytest.ini_options]
+addopts = "--pyargs optax --numprocesses auto"
+
+[tool.pylint.main]
+ignore-patterns = [
+    ".*.pyi",
+]
+ignore = [
+    "test_venv",
+]
+
+[tool.pylint.messages_control]
```

Comment:
Just to understand: why did you disable all these errors?

Just for you to know: each time a PR does not comply to pylint correctly, we have to patch it internally which slows down the process of migration. So as annoying as pylint may be, we have no other choice to comply with it normally.

I agree that actually the code should be fully cleaned to comply with the usual formattings. If that's the reason for you to disable all these messages let me know. If it's because you are using flake8, that's great.